templates/Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
+web: bundle exec rails serve Puma -p $PORT -c ./config/puma.rb
```

Comment:
Why `rails serve Puma` rather than `bundle exec puma` per https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server?